client/src/app/containers/Game/index.jsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+
+import Grid from '@material-ui/core/Grid';
+import RefreshIcon from '@material-ui/icons/Refresh';
+import IconButton from '@material-ui/core/IconButton';
```

Comment:
Again, I thought eslint would group `import React from 'react';`, then the external imports, then the local imports, with a new line in between each? e.g.:
https://github.com/joshprzybyszewski/cribbage/blob/c22b7a2df587adfbd59f1e97df98ec78a5fd725c/client/src/app/containers/LeftDrawer/index.jsx#L1-L18